ruby/basic_ruby/loops.md

```diff
@@ -147,23 +148,25 @@ If you need to step through a series of numbers (or even letters) within a speci
 <div class="lesson-content__panel" markdown="1">
 
 1. Read the [Loops & Iterators chapter](https://launchschool.com/books/ruby/read/loops_iterators) of LaunchSchool's *Introduction to Programming With Ruby*. We recommend skipping the section on [recursion](https://launchschool.com/books/ruby/read/loops_iterators#recursion) since recursion will be covered later on.
```

Comment:
```suggestion
1. Read the [Loops & Iterators chapter](https://launchschool.com/books/ruby/read/loops_iterators) of LaunchSchool's *Introduction to Programming With Ruby*. We recommend skipping the section on recursion since recursion will be covered later on.
```
Planned removal of a redundant link.